packages/react/package.json

```diff
@@ -45,7 +45,7 @@
   },
   "dependencies": {
     "@fontsource-variable/roboto-flex": "5.0.8",
-    "@fontsource/material-icons": "5.0.11",
+    "@fontsource/material-icons-outlined": "5.0.11",
```

Comment:
Are we sure about the fact to fully remove `@fontsource/material-icons`. I checked the bolt icon used in the storybook, It appears like before but honestly I do not understand why exactly so I'm not confident with this diff.

Reply:
As I understand, the default `@fontsource/material-icons` is equivalent of the "filled" variant. And according to https://fonts.google.com/icons?icon.query=bolt&icon.set=Material+Icons&icon.style=Outlined, the bolt remain the same across the two variant. 

<img width="556" alt="Capture d’écran 2024-01-08 à 09 42 19" src="https://github.com/openfun/cunningham/assets/9628870/627a2f60-c32f-4fa6-a04e-6ed9309d706e">

<img width="484" alt="Capture d’écran 2024-01-08 à 09 42 23" src="https://github.com/openfun/cunningham/assets/9628870/825426ab-2037-44c1-9e31-aac01a7cbd5e">